### PR TITLE
Add Not Applicable Option to Target Pop. Choice List

### PR DESCRIPTION
**Pull request by keeysnc**
### Description
Adding "Not Applicable" option to the target populations choice list. 

Screenshot 2023-12-11 at 4 54 38 PM

### Related ticket(s)
[CMDCT-](https://jiraent.cms.gov/browse/CMDCT-3109)

---
### How to test
1. Create a report 
2. Go to 'State- or Territory-Specific Initiatives' and create and edit a initiative
3. Edit 'I. Define initiative'
4. In the Target population(s) choice list you should see the 'Not applicable' option.

### Important updates

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/utils/forms/forms.ts [hunk 1]
```diff
@@ -311,8 +320,18 @@ export const updateRenderFields = (
...
+  const notApplicablePopulation = {
+    id: "3Nc13O5GHA6Hc4KheO5FMSD2",
+    transitionBenchmarks_targetPopulationName: "Not applicable",
+    type: "targetPopulations",
+    isRequired: false,
+  };
+
+  const filteredTargetPopulations = new Array(
+    ...removeNotApplicablePopulations(targetPopulations),
+    notApplicablePopulation
```
Comment (gmrabian) [inline services/ui-src/src/utils/forms/forms.ts:333]: comment: these two names are unfortunate together (remove not applicable, then next line you add "notApplicable") but I understand what is happening

**Review by gmrabian — APPROVED** [review]
looks good locally

File: services/ui-src/src/utils/forms/forms.ts (hunk 1 above)
Comment (keeysnc) [reply services/ui-src/src/utils/forms/forms.ts]: ha yea good point. I just updated it to notApplicableOption.

**Review by britt-mo — APPROVED** [review]
tested locally, works according to MVP requirements

**Review by gmrabian — APPROVED** [review]

**Pull request merged**